packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.html

```diff
@@ -25,7 +25,7 @@
         <template if:false={isStandardFacet}>
           <span class="slds-truncate slds-var-m-right_xx-small slds-var-m-top_xxx-small slds-text-body_small nowrap facet__value-text facet__value_idle">{formattedFacetValue}</span>
         </template>
-        <span class="nowrap slds-text-body_small facet__number-of-results">({numberOfResults})</span>
+        <span class="nowrap slds-text-body_small slds-var-m-top_xxx-small facet__number-of-results">({numberOfResults})</span>
```

Comment:
`slds-var-m-top_xxx-small` does not seem to be necessary.

Reply:
I disagree. 😅 Otherwise is is not vertically centered

with
<img width="566" alt="Screen Shot 2021-11-10 at 10 55 25 AM" src="https://user-images.githubusercontent.com/16785453/141147068-6b3cd6c2-ef33-4df9-8844-18e0905586c9.png">

without
<img width="588" alt="Screen Shot 2021-11-10 at 10 55 03 AM" src="https://user-images.githubusercontent.com/16785453/141147094-a45ed9e1-4db5-4f1d-ad0a-775454b7ee71.png">

Reply:
:sweat_smile: You're right, the difference is visible on the screenshots. It was to subtle to notice in my browser.